core/src/main/resources/META-INF/fabric8/kind-fileindicator-mapping-default.adoc

```diff
@@ -0,0 +1,102 @@
+|===
+|Kind |Filename Type
+
+|ConfigMap
+a|`cm`, `configmap`
+
+|CronJob
+a|`cronjob`, `cj`
+
+|ClusterRole
+a|`cr`, `crole`, `clusterrole`
+
+|CustomResourceDefinition
+a|`crd`
+
+|ClusterRoleBinding
+a|`crb`, `clusterrb`
+
+|Deployment
+a|`deployment`
+
+|ImageStream
+a|`is`
+
+|ImageStreamTag
+a|`istag`
+
+|Job
+a|`job`
+
+|LimitRange
+a|`lr`, `limitrange`
+
+|Namespace
+a|`ns`, `clusterrb`
```

Comment:
hmm. not really sure what "clusterrb" should be and why we allow a namespace resoure at all. I know its not part of the PR, I just thinking out loud ;-)